src/core/lib/transport/method_config.h

```diff
@@ -37,7 +37,7 @@
 #include <grpc/impl/codegen/gpr_types.h>
```

Comment:
The method_config.h and method_config.c files were removed in #8617, but it looks like they were accidentally re-added in #8705.  They're not used anywhere, so they can be removed.